src/panels/energy/ha-panel-energy.ts

```diff
@@ -30,12 +32,27 @@ import {
 import { fileDownload } from "../../util/file_download";
 import type { StatisticValue } from "../../data/recorder";
 
+export const DEFAULT_ENERGY_COLLECTION_KEY = "energy_dashboard";
+
 const ENERGY_LOVELACE_CONFIG: LovelaceConfig = {
   views: [
+    {
+      strategy: {
+        type: "energy-overview",
+        collection_key: DEFAULT_ENERGY_COLLECTION_KEY,
+      },
+    },
     {
       strategy: {
         type: "energy",
```

Comment:
Is this kept for backwards compatibility? Or should we just rename it to:
```suggestion
        type: "energy-electricity",
```

Reply:
Renamed